caterva2/api.py

```diff
@@ -589,16 +777,36 @@ def __iter__(self):
 
         Examples
         --------
-        >>> root = cat2.Root('@public')
+        >>> import caterva2 as cat2
+        >>> root = cat2.Root('example', 'https://demo.caterva2.net')
         >>> for file in root:
         ...     print(file)
-        @public/ds-1d.b2nd
-        @public/ds-2d.b2nd
-
+        ds-2d-fields.b2nd
+        lung-jpeg2000_10x.b2nd
+        ds-1d-fields.b2nd
+        ds-1d.b2nd
+        ds-hello.b2frame
+        ds-sc-attr.b2nd
+        README.md
+        ds-1d-b.b2nd
+        tomo-guess-test.b2nd
+        dir2/ds-4d.b2nd
+        dir1/ds-3d.b2nd
+        dir1/ds-2d.b2nd
```

Comment:
I'd say that it would be better to list the complete path, instead of only the part below the root.  What do you think @martaiborra ?

Reply:
Unfortunately, most of the code is meant to remove the root from the path, so this will be a major change.  Better letting that as-is for now.

Reply:
FWIW, I think this way of listing avoiding the prefix (and not only the root, as I was supposing) is made on purpose, and consistent with the `ls` tool on unix:

```
> ls -R caterva2/clients/                                                           (base)
__pycache__/ cli.py       cli.py.orig  tbrowser.py

caterva2/clients//__pycache__:
cli.cpython-312.pyc
```